plugin/api.go

```diff
@@ -474,7 +474,17 @@ type API interface {
 	// Minimum server version: 5.6
 	KVSetWithExpiry(key string, value []byte, expireInSeconds int64) *model.AppError
 
-	// KVGet retrieves a value based on the key, unique per plugin. Returns nil for non-existent keys.
+	// KVCompareAndSetWithExpiry will update a key-value pair, with an expiry time
```

Comment:
I see the original doc of `KVCompareAndSet` has already been updated. Since it'll be harder to keep docs in sync for both `KVCompareAndSet` and `KVCompareAndSetWithExpiry` I would suggest to just refer them to `KVCompareAndSet` for better maintainability. Something like:

```
// KVCompareAndSetWithExpiry same as KVCompareAndSet except with an expiry time.
```

Reply:
Done.